pkg/util/converter/prom.go

```diff
@@ -925,6 +946,182 @@ func readStream(iter *jsonitere.Iterator) backend.DataResponse {
 	return rsp
 }
 
```

Comment:
These files and these changes could use some unit testing.

https://github.com/grafana/grafana/pull/76350#issuecomment-1757589431

Reply:
I created https://github.com/grafana/grafana/issues/77349, also to cover your previous point. I think the coverage is mostly going down because `prom.go` is mostly tested with these golden JSON files, where coverage does not seem to work properly:

![image](https://github.com/grafana/grafana/assets/8092184/e3bc5d5b-f7f6-4426-bb60-7d7abcc55db2)